text/0000-local-snapshot-file-format/0000-local-snapshot-file-format.md

```diff
@@ -0,0 +1,186 @@
+                                                        </td>
+                                                    </tr>
+                                                    <tr>
+                                                        <td>Address</td>
+                                                        <td>ByteArray[49]</td>
+                                                        <td>The T5B1 encoded WOTS address.</td>
+                                                    </tr>
+                                                </table>
+                                            </details>
+                                            <details>
+                                                <summary>Ed25519 Address</summary>
+                                                <table>
+                                                    <tr>
+                                                        <td><b>Name<b></td>
+                                                        <td><b>Type</b></td>
+                                                        <td><b>Description</b></td>
+                                                    </tr>
+                                                    <tr>
+                                                        <td>Address Type</td>
+                                                        <td>byte/varint</td>
+                                                        <td>
+                                                            Set to <strong>value 1</strong> to denote an <i>Ed25519 Address</i>.
+                                                        </td>
+                                                    </tr>
+                                                    <tr>
+                                                        <td>Address</td>
+                                                        <td>ByteArray[32]</td>
+                                                        <td>The raw bytes of the Ed25519 address which is a BLAKE2b-256 hash of the Ed25519 public key.</td>
+                                                    </tr>
+                                                </table>
+                                            </details>
+                                        </td>
+                                    </tr>
+                                    <tr>
+                                        <td>Value</td>
+                                        <td>uint64</td>
+                                        <td>The output value.</td>
+                                    </tr>
+                                </table>
+                            </details>
+                        </td>
+                    </tr>
+                </table>
+            </details>
+        </td>
+    </tr>
+    <tr>
+        <td>Integrity hash</td>
+        <td>ByteArray[32]</td>
+        <td>
+        The SHA256 hash of all the previous fields.
```

Comment:
Why introducing another hash type instead of using `BLAKE2b-256`?

Reply:
I was wondering about the same thing :thinking: 
What is actually the purpose of this?

If it's about authenticity it should be a signature, shouldn't it? 
If it is just about data integrity, I think this should be part of the second layer. Exactly like the compression.
Probably, I would just remove it and lift that functionality higher.

Reply:
Will be removed.